orchagent/portsorch.h

```diff
@@ -110,7 +110,8 @@ class PortsOrch : public Orch, public Subject
     Port m_cpuPort;
     // TODO: Add Bridge/Vlan class
     sai_object_id_t m_default1QBridge;
-    sai_object_id_t m_defaultVlan;
+    sai_object_id_t m_defaultVlan_ObjId;
```

Comment:
Thanks for submitting the PR. The logic looks valid to me. Could you modify the type of the current m_defaultVlan to type `Port`? Then you could use `m_vlan_info` to store both the oid and the id of the VLAN. This would prevent having more private variables.